ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -108,6 +132,7 @@ pub(crate) struct GlobalDeclIdx(usize);
 index!(GlobalDeclIdx);
 
 /// An index into [FuncType::arg_ty_idxs].
+/// ^ FIXME: no it's not! But it should be!
```

Comment:
I believe it's currently used for both looking up an argument type in the `FuncType` as well as looking up an argument passed into a function call.

Reply:
The problem is that the vector isn't a `TiVec` yet. I will fix this in a follow-up PR.